rancher/rancher-compose/scheduling/index.md

```diff
@@ -91,6 +91,26 @@ labels:
   io.rancher.scheduler.affinity:host_label_soft_ne: key4=value4
 ```
 
+**Automatically Applied Host Labels**
+
+In v0.46.0+, Rancher has automatically added host labels to allow for easier scheduling. 
```

Comment:
In v0.46.0+, Rancher automatically adds host labels to allow for easier scheduling.